<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bandmerch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="vlayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Artist/Label Name </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="label_edit"/>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Merch Type</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="3" column="3">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>Hat</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_15">
          <property name="text">
           <string>Cassette</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="checkBox_12">
          <property name="text">
           <string>Bag</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QCheckBox" name="checkBox_13">
          <property name="text">
           <string>Other Apparel</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>T-Shirt/Shirt</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_8">
          <property name="text">
           <string>Sheet Music</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="checkBox_16">
          <property name="text">
           <string>Compact Disc (CD)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="checkBox_11">
          <property name="text">
           <string>Ticket</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Sweater/Hoodie</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="checkBox_7">
          <property name="text">
           <string>USB Flash Drive</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="checkBox_14">
          <property name="text">
           <string>Book/Magazine</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox_10">
          <property name="text">
           <string>Record/Vinyl</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="checkBox_6">
          <property name="text">
           <string>DVD</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_9">
          <property name="text">
           <string>Poster/Print</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>T-Shirt/Apparel</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Other</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">type_btns</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Sort By</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </item>
        <item>
         <widget class="QComboBox" name="sort_combo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Nothing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Artist</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Price</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Title</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Type</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Ascending</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">sort_btns</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Descending</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">sort_btns</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Other Options</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="include_sold_out_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Include Sold Out</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="gen_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="type_btns">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="sort_btns"/>
 </buttongroups>
</ui>
